spring-5-reactive-client/src/test/java/com/baeldung/reactive/simultaneouswebclient/ClientIntegrationTest.java

```diff
@@ -0,0 +1,71 @@
+package com.baeldung.reactive.simultaneouswebclient;
+
+import org.junit.Test;
+import org.junit.Before;
+import org.junit.After;
+import org.junit.runner.RunWith;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+import com.github.tomakehurst.wiremock.WireMockServer;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+import static com.github.tomakehurst.wiremock.core.WireMockConfiguration.wireMockConfig;
+import static com.github.tomakehurst.wiremock.client.WireMock.*;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.assertEquals;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+public class ClientIntegrationTest {
+
```

Comment:
This class also needs formatting correctly using the eclipse formatter we have.
https://github.com/eugenp/tutorials/blob/master/eclipse/formatter.xml
Thanks.

Reply:
fixed